### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by cs-abhinav-goel**
Initial Draft

### CODE REVIEW

File: widget/view.controller.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  var currentTheme = $rootScope.theme.id;
+  $scope.entity = FormEntityService.get();
+  $scope.pickListValue = $scope.entity['originalData'][config.picklistItem] ? $scope.entity['originalData'][config.picklistItem]['itemValue'] : '';
+
+  function init() {
+    widgetWSSubscribe();
+    getPicklistValues();
+    $scope.activeItemImage = currentTheme === 'light' ? 'widgets/installed/infoGraphics-1.0.0/images/chevron_inactive_arrow.png' : 'widgets/installed/infoGraphics-1.0.0/images/chevron_active_arrow.png';
+    $scope.inactiveItemImage = currentTheme === 'light' ? 'widgets/installed/infoGraphics-1.0.0/images/chevron_active_arrow.png' : 'widgets/installed/infoGraphics-1.0.0/images/chevron_inactive_arrow.png';
+  }
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js:25]: import & use widgetBasePath 
widgetBasePath +'images/chevron_inactive_arrow.png'

**Review by cs-pranali-chavan — CHANGES_REQUESTED** [review]
use widgetBasePath for version updates

File: widget/info.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "infoGraphics",
+    "title": "Info Graphics",
```
Comment (cs-suyog-jinturkar) [inline widget/info.json]: snapshots will be added here as well which will show edit and view panel screenshots

File: widget/view.controller.js (hunk 1 above)
Comment (cs-abhinav-goel) [reply widget/view.controller.js:25]: Done

**Pull request merged**